cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -85,3 +85,14 @@ resource "aws_db_instance" "civiform" {
   skip_final_snapshot     = true
   backup_retention_period = var.postgres_backup_retention_days
 }
+
+module "email_service" {
+  for_each = toset([
+    var.sender_email_address,
+    var.staging_applicant_notification_mailing_list,
+    var.staging_ti_notification_mailing_list,
+    var.staging_program_admin_notification_mailing_list
+  ])
+  source               = "../../modules/ses"
+  sender_email_address = each.key
```

Comment:
question (non-blocking): how does `for_each` and `each.key` work here?

Reply:
I copied it from azure setup =) https://www.terraform.io/language/meta-arguments/for_each
It will create a separate resource for each element.